hwbench/environment/block_devices.py

```diff
@@ -79,8 +79,8 @@ def get_udev_attributes(self) -> dict[str, Any]:
         for attribute in attributes.available_attributes:
             # The casting below is to avoid breaking struct to json. b"" cant be serialized has to be cast as string
             dumped[attribute] = (
-                str(attributes.get(attribute))
-                if type(attributes.get(attribute)) is bytes
+                attributes.get(attribute).decode()
```

Comment:
Don't you need to specify an encoding?
Can an attribute be something other than a `bytes` ?

Reply:
I do not, by default it is utf-_ anyway.
Yes, some are null (why? no se)

Reply:
Why are some null?

Reply:
Ack